members/OHL000283.yaml

```diff
@@ -13,6 +13,8 @@ contact_form:
           selector: "#tbxLastName"
           value: $NAME_LAST
           required: true
+          options:
+            blacklist: "'"
```

Comment:
Do we need to escape this for the yaml to parse?

Reply:
I don't think so. Tested it a few times and confirmed that the `blacklist` step worked as intended